enos/enos-scenario-e2e-aws-windows.hcl

```diff
@@ -0,0 +1,312 @@
+  providers = [
+    provider.aws.default,
+    provider.enos.default
+  ]
+
+  matrix {
+    builder    = ["local", "crt"]
+    client     = ["win10", "win11"]
+    rdp_server = ["2022", "2025"]
+  }
+
+  locals {
+    aws_ssh_private_key_path  = abspath(var.aws_ssh_private_key_path)
+    boundary_install_dir      = abspath(var.boundary_install_dir)
+    local_boundary_dir        = var.local_boundary_dir != null ? abspath(var.local_boundary_dir) : null
+    local_boundary_src_dir    = var.local_boundary_src_dir != null ? abspath(var.local_boundary_src_dir) : null
+    local_boundary_ui_src_dir = var.local_boundary_ui_src_dir != null ? abspath(var.local_boundary_ui_src_dir) : null
+    boundary_zip_windows      = abspath(var.boundary_zip_windows)
+    boundary_license_path     = abspath(var.boundary_license_path != null ? var.boundary_license_path : joinpath(path.root, "./support/boundary.hclic"))
+    vault_license_path        = abspath(var.vault_license_path != null ? var.vault_license_path : joinpath(path.root, "./support/vault.hclic"))
+
+    build_path = {
+      "local" = "/tmp",
+      "crt"   = var.crt_bundle_path == null ? null : abspath(var.crt_bundle_path)
+    }
+
+    tags = merge({
+      "Project Name" : var.project_name
+      "Project" : "Enos",
+      "Environment" : "ci"
+    }, var.tags)
+
+    collocated_tag = "collocated"
+  }
+
+  step "find_azs" {
+    module = module.aws_az_finder
+
+    variables {
+      instance_type = [
+        var.worker_instance_type,
+        var.controller_instance_type
+      ]
+    }
+  }
+
+  step "read_boundary_license" {
+    module = module.read_license
+
+    variables {
+      license_path = local.boundary_license_path
+    }
+  }
+
+  step "read_vault_license" {
+    module = module.read_license
+
+    variables {
+      license_path = local.vault_license_path
+    }
+  }
+
+  step "create_db_password" {
+    module = module.random_stringifier
+  }
+
+  step "build_boundary" {
+    module = matrix.builder == "crt" ? module.build_crt : module.build_local
+
+    variables {
+      path    = local.build_path[matrix.builder]
+      edition = var.boundary_edition
+    }
+  }
+
+  step "build_boundary_windows" {
+    module = matrix.builder == "crt" ? module.build_crt : module.build_local
+
+    depends_on = [
+      step.build_boundary
+    ]
+
+    variables {
+      path          = local.build_path[matrix.builder]
+      edition       = var.boundary_edition
+      goos          = "windows"
+      build_target  = "build"
+      artifact_name = "boundary_windows"
+      binary_name   = "boundary.exe"
+    }
+  }
+
+  step "create_base_infra" {
+    module = module.aws_vpc_ipv6
+
+    depends_on = [
+      step.find_azs,
+    ]
+
+    variables {
+      availability_zones = step.find_azs.availability_zones
+      common_tags        = local.tags
+    }
+  }
+
+  step "create_windows_client" {
+    module = module.aws_client_windows
+
+    depends_on = [
+      step.create_base_infra,
+    ]
```

Comment:
We'll want to add a dependency to `step.build_boundary_windows` here. It *should* be implicit since we use it in the variables, but doesn't hurt to be explicit here on the dependency.